packages/odyssey-react-mui/src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import { Alert, Link } from "../..";
+
+export interface BannerProps {
+  severity?: AlertColor;
+  role?: string;
+  children: any;
+  onClose?: any;
```

Comment:
Rather than expose `onClose` directly, should we include the JS for dismissal and change this to a boolean called `isDismissible`? That way, we'll know the behavior is consistent.